R/aaa-auto.R

```diff
@@ -1240,6 +1240,21 @@ reciprocity_impl <- function(graph, ignore.loops=TRUE, mode=c("default", "ratio"
   res
 }
 
+maxdegree_impl <- function(graph, ..., v=V(graph), mode=c("all", "out", "in", "total"), loops=TRUE) {
+  # Argument checks
+  check_dots_empty()
+  ensure_igraph(graph)
+  v <- as_igraph_vs(graph, v)
+  mode <- switch(igraph.match.arg(mode), "out"=1L, "in"=2L, "all"=3L, "total"=3L)
```

Comment:
how can we fix the spacing of the autogenerated code, should it be run through the styler R package? I'd expect spaces around equal signs for instance.